scene/gui/rich_text_label.cpp

```diff
@@ -497,7 +497,7 @@ int RichTextLabel::_process_line(ItemFrame *p_frame, const Vector2 &p_ofs, int &
 							Color uc = color;
 							uc.a *= 0.5;
 							int uy = y + lh / 2 - line_descent + 2;
-							float strikethrough_width = 1.0;
+							float strikethrough_width = font->get_underline_thickness();
 #ifdef TOOLS_ENABLED
 							strikethrough_width *= EDSCALE;
```

Comment:
Is the `EDSCALE` factor still needed now that we retrieve this information from the font? Or would it scale automatically together with the font?